ocs_ci/ocs/tests/fio.output

```diff
@@ -0,0 +1,297 @@
+{
```

Comment:
please move this into templates, that is still the right place along with other yaml's.

Reply:
This is not a template. Moreover it's part of the unit test.